keras/engine/training.py

```diff
@@ -829,6 +832,53 @@ def _get_callback_model(self):
             return self.callback_model
         return self
 
+    def _validate_or_infer_batch_size(self, batch_size, steps, x):
+        """Validates that the `batch_size` provided is consistent with InputLayer.
+
+        It's possible that the user specified a static batch size in their
+        InputLayer. If so, this method checks the provided `batch_size` and `x`
+        arguments are consistent with this static batch size. Also, if
+        `batch_size` is `None`, this method will attempt to infer the batch size
+        from the static batch size of the InputLayer. Lastly, ValueError will be
+        raised if `x` is a tf.data.Dataset and `batch_size` is specified as we
```

Comment:
This is not relevant to Keras (which does not accept tf.data.Dataset instances as inputs)

Reply:
I updated this to to only refer to `Sequences` and generators.